components/board/upboard/data.go

```diff
@@ -15,40 +15,41 @@ var boardInfoMappings = map[string]genericlinux.BoardInformation{
 				GPIOChipSysFSDir: path to the directory of a chip. Can be found from the output of gpiodetect
 			*/
 			// GPIO pin definition
-			{Name: "29", Ngpio: 78, LineNumber: 73, PwmChipSysfsDir: "", PwmID: -1},
-			{Name: "31", Ngpio: 77, LineNumber: 46, PwmChipSysfsDir: "", PwmID: -1},
-			{Name: "18", Ngpio: 77, LineNumber: 48, PwmChipSysfsDir: "", PwmID: -1},
-			{Name: "22", Ngpio: 77, LineNumber: 45, PwmChipSysfsDir: "", PwmID: -1},
-			{Name: "37", Ngpio: 77, LineNumber: 46, PwmChipSysfsDir: "", PwmID: -1},
-			{Name: "35", Ngpio: 47, LineNumber: 17, PwmChipSysfsDir: "", PwmID: -1},
-			{Name: "13", Ngpio: 77, LineNumber: 75, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "29", Ngpio: 28, LineNumber: 5, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "31", Ngpio: 28, LineNumber: 6, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "18", Ngpio: 28, LineNumber: 24, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "22", Ngpio: 28, LineNumber: 25, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "37", Ngpio: 28, LineNumber: 26, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "35", Ngpio: 28, LineNumber: 19, PwmChipSysfsDir: "", PwmID: -1},
```

Comment:
Low-priority suggestion: sort each paragraph by name. Consider getting rid of the paragraphs and sorting all pins by name. That will make it easier for people to find the definitions for specific pins.